specutils/tests/test_loaders.py

```diff
@@ -15,15 +15,11 @@
 from astropy.utils.data import get_pkg_data_filename
 
 from .. import Spectrum1D
-from .conftest import remote_data_path
+from .conftest import remote_data_path, remote_access
```

Comment:
Doesn't `pytest` automatically make these available within the test module namespace without an import?

Reply:
You are correct! Except for the remote_access function. That needs to be imported.